src/config.cpp

```diff
@@ -157,23 +181,34 @@ void parse_program_options(int argc, const char** argv)
         std::exit(0);
     }
 
-    if (trace)
+    if (quiet && verbosity.count != 0)
     {
-        lo2s::logging::set_min_severity_level(nitro::log::severity_level::trace);
-        lo2s::Log::trace() << "Enabling log-level 'trace'";
+        lo2s::Log::warn() << "Cannot be quiet and verbose at the same time. Refusing to be quiet.";
+        quiet = false;
     }
-    else if (debug)
-    {
-        lo2s::logging::set_min_severity_level(nitro::log::severity_level::debug);
-        lo2s::Log::debug() << "Enabling log-level 'debug'";
-    }
-    else if (quiet)
+
+    if (quiet)
     {
         lo2s::logging::set_min_severity_level(nitro::log::severity_level::warn);
     }
     else
     {
-        lo2s::logging::set_min_severity_level(nitro::log::severity_level::info);
+        using sl = nitro::log::severity_level;
+        switch (verbosity.count)
+        {
+        case 0:
+            lo2s::logging::set_min_severity_level(sl::warn);
```

Comment:
At least there should be a way to get `info`, but I'm not sure if it should be default.
How about:

. | .
------------ | -------------
`-q` | error
`(default)` | warn
`-v` | info
`-vv` | debug
`-vvv` | trace